lib/krb5/krb5.conf.5

```diff
@@ -925,6 +925,70 @@ The maximim number of dots in a name matched via
 derived key namespaces.
 .El
 .Pp
+.It Li [hdb]
+.Bl -tag -width "xxx" -offset indent
+.It Li db-dir = Va path
+This parameter defines a directory that can contain:
+.Bl -tag -width "xxx" -offset indent
+.It Va kdc.conf
+A configuration file with the same format as krb5.conf that will
+be included.
```

Comment:
Clarify that any `kdc.conf` is a) included as if by the included directive, and b) only by programs like kadmin, kadmind, kdc, etc.